sandbox_container/installer.sh

```
@@ -0,0 +1,106 @@
+
+echo -e "${GREEN}ClaudeCode 開発環境のセットアップを開始します...${NC}"
+
+# --- 1. ディレクトリの作成 ---
+PROJECT_DIR="my-claude-dev"
+if [ -d "$PROJECT_DIR" ]; then
+  echo -e "${YELLOW}警告: ディレクトリ '$PROJECT_DIR' は既に存在します。${NC}"
+else
+  mkdir "$PROJECT_DIR"
+  echo "作業ディレクトリを作成しました: $PROJECT_DIR"
+fi
+cd "$PROJECT_DIR"
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)